pkg/registry/core/secret/strategy_test.go

```diff
@@ -28,6 +28,9 @@ import (
 	api "k8s.io/kubernetes/pkg/apis/core"
 
 	// install all api groups for testing
+	"fmt"
```

Comment:
Move up, to the group where all golang imports are. The below imports should go up to k8s imports as well.